isis/src/base/objs/StretchBlob/StretchBlob.cpp

```diff
@@ -36,7 +36,16 @@ namespace Isis {
    *
    * @param name Name to use for Stretch
    */
-  StretchBlob::StretchBlob(QString name) : Blob(name, "Stretch"), m_stretch(name) {
+  StretchBlob::StretchBlob(QString name) {
+    m_stretch = CubeStretch(name, "Stretch");
```

Comment:
Similar to comment above, I'd say either just use `CubeStretch(name)` or at least use the same default argument of "Default" since the second argument to the `CubeStretch` constructor is the type of the stretch, and that's used in the GUI StretchTool to determine what to display.